### PR TITLE
Add AI/BI dashboard example

### PR DESCRIPTION
**Pull request by pietern**
This example demonstrates how to model an AI/BI dashboard in a Databricks Asset Bundle.

It includes and deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace.

For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/en/dashboards/index.html#dashboards).

**This requires Databricks CLI v0.232.0 (to be released on Wed Oct 30) or above.**

### CODE REVIEW

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Dashboard for NYC Taxi Trip Analysis
+
+This example demonstrates how to model an AI/BI dashboard in a Databricks Asset Bundle.
+
+It includes and deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace.
+
+For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/en/dashboards/index.html#dashboards).
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:7]:
```suggestion
For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/en/dashboards/index.html).
```
Anchor is not needed as this is the main dashboards article.

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Prerequisites
+
+* Databricks CLI v0.232.0 or above
+
+## Usage
+
+First:
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:15]:
```suggestion
Modify `databricks.yml`:
```

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* Update the `host` field under `workspace` in `databricks.yml` to the Databricks workspace you wish to deploy to.
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:16]:
```suggestion
* Update the `host` field under `workspace` to the Databricks workspace to deploy to.
```

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+* Update the `warehouse` field under the `warehouse_id` variable to match the SQL warehouse you wish to use.
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:17]:
```suggestion
* Update the `warehouse` field under `warehouse_id` to the name of the SQL warehouse to use.
```

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+Run `databricks bundle deploy` to deploy the dashboard.
+
+Run `databricks bundle open` to open the deployed dashboard in your browser.
+
+Alternatively, run `databricks bundle summary` to display its URL.
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:23]:
```suggestion
Run `databricks bundle open` to navigate to the deployed dashboard in your browser. Alternatively, run `databricks bundle summary` to display its URL.
```

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+It includes and deploys the sample __NYC Taxi Trip Analysis__ dashboard to a Databricks workspace.
+
+For more information about AI/BI dashboards, please refer to the [documentation](https://docs.databricks.com/en/dashboards/index.html#dashboards).
+
+## Prerequisites
+
+* Databricks CLI v0.232.0 or above
+
+## Usage
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:13]: Should the steps in here be numbered?

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+Run `databricks bundle open` to open the deployed dashboard in your browser.
+
+Alternatively, run `databricks bundle summary` to display its URL.
+
+### Visual modification
+
+Dashboards are inherently visual, so you may want to use the web interface to modify the dashboard.
+Any modifications that are done through the web interface won't automatically be reflected in the `.lvdash.json` file.
+You can, however, run a dedicated bundle command to retrieve these changes and update your `.lvdash.json` file.
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:29]:
```suggestion
You can use the Databricks UI to modify the dashboard, but any modifications made through the UI will not be applied to the bundle `.lvdash.json` file unless you run `bundle generate` or `bundle deploy --force`.
```

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+To retrieve the updated `.lvdash.json` file, run:
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:31]:
```suggestion
To update the local bundle `.lvdash.json` file, run:
```

Comment (juliacrawf-db) [reply knowledge_base/dashboard_nyc_taxi/README.md:31]: (Hmm....maybe original is better...)

File: knowledge_base/dashboard_nyc_taxi/README.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+To continuously poll and retrieve the updated `.lvdash.json` file when it changes, run:
+
+```sh
+databricks bundle generate dashboard --resource nyc_taxi_trip_analysis --force --watch
+```
+
+Any remote modifications of a dashboard are noticed by the `deploy` command and require
+you to explicitly acknowledge that remote changes are overwritten by local changes.
```
Comment (juliacrawf-db) [inline knowledge_base/dashboard_nyc_taxi/README.md:44]:
```suggestion
you to explicitly acknowledge that remote changes can be overwritten by local changes.
```

File: knowledge_base/dashboard_nyc_taxi/README.md (hunk 6 above)
Comment (pietern) [reply knowledge_base/dashboard_nyc_taxi/README.md:13]: We could but then should apply it in the other examples in the knowledge base as well.

What do you typically convey with numbering over plain enumeration?

**Pull request merged**